lib/lambda.ts

```diff
@@ -60,7 +61,7 @@ export const workTests = async (
     }
 
     const run = async () => {
-      const tab = await prepareChrome({ sessionId: '' })
+      const tab = await prepareChrome({ sessionId: opts.sessionId })
```

Comment:
The linting PR broke this bit to get the types work. It was apparently a core code path but there was never an new upload of lambda code to catch this issue